app/src/main/java/com/kahuis/repairtool/data/remote/dto/RepairDetailDto.kt

```diff
@@ -0,0 +1,31 @@
+package com.kahuis.repairtool.data.remote.dto
+
+import com.kahuis.repairtool.domain.model.RepairDetail
+
+data class RepairDetailDto(
+    val id: String,
+    val title: String,
+    val description: String,
+    val reporter_id: String,
```

Comment:
How are you going to deal with relationships? This reporter can for example be included. The object will then be injected under the `reporter` key.